internal/source/vmware_linux.go

```diff
@@ -8,6 +8,7 @@ import (
 	"net/url"
 	"time"
 
+	log "github.com/sirupsen/logrus"
```

Comment:
why logrus? I thought we were using slog usually.

Reply:
We still use logrus on the worker side for the time being. I thought it best for this PR to just keep the current format rather than instantiate both slog and logrus, or make this PR longer by removing logrus. 

 I've filed [an issue](https://github.com/FuturFusion/migration-manager/issues/138) to move all of these to slog.

Reply:
Ah, interesting, not sure why we didn't move that over at the same time as the rest :)

I know we had a couple of spots where an external package uses logrus, but that's why we have the SlogBackedLogrus stuff.